modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java

```diff
@@ -249,7 +252,9 @@ protected String getDDMFormFieldValue(
 		stream.map(
 			this::getNontransientDDMFormFieldsMap
 		).forEach(
-			ddmFormFields::putAll
+			map -> map.forEach(
+				(key, ddmFormField) -> ddmFormFields.putIfAbsent(
+					key, ddmFormField))
```

Comment:
@rodrigopaulino, can you please clarify why this change was necessary? Which problem is it exactly addressing?

Reply:
This is the verification we've talked offline that is necessary to maintain the latest version field labels. If we simply put the same field in that order, the label exported would be the first version's, and that is the original bug reported on 7.1.x.

Reply:
Makes sense to me! Thanks for the clarification, @rodrigopaulino. 😉